pkg/kubectl/cmd/run.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const (
 	run_long = `Create and run a particular image, possibly replicated.
-Creates a replication controller to manage the created container(s).`
+Creates a deployment or job to manage the created container(s).`
```

Comment:
We cannot remove those without deprecating them and notifying users.  I
suspect we do not want to remove them before 1.2


On Nov 14, 2015, at 9:16 AM, Michail Kargakis <notifications@github.com>
wrote:

In pkg/kubectl/cmd/run.go
<https://github.com/kubernetes/kubernetes/pull/17195#discussion_r44858944>:

> @@ -34,7 +35,7 @@ import (
>
>  const (
>  	run_long = `Create and run a particular image, possibly replicated.
> -Creates a replication controller to manage the created container(s).`
> +Creates a deployment or job to manage the created container(s).`

hm, I see you have removed the rc and pod generators below. Isn't this
going to break people?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/17195/files#r44858944>.